index.php

```diff
@@ -27,6 +27,46 @@
    exit;
 }
 
+//function to show warnings
+function warning($msg) { 
+	$script = "<script type='text/javascript'>$('#warning').fadeIn(1000);</script>";
+	$script .= "<script type='text/javascript'>$('#warning').html('<strong>Warning!</strong> $msg');</script>";
+	return $script;
+}
+
+//check if wol application (either wol or awake) is installed and set wol application to be used (e.g. for use with alpine & awake instead of wol)
+$wol_apps = [ "wakeonlan", "awake" ];//could be extended
+foreach ($wol_apps as $app) {
+  if (exec("command -v ".$app)<>"")
+  {
+    $wol = $app;//set the first application from the array that returns a non empty string as the wol application to be used
+    break;
+  }
+  if ($app === end($wol_apps))
+  {
+    $apps = implode(', ', $wol_apps);
+    $script = warning("unable to find any wol app, been looking for: <strong>".$apps."</strong> (one is necessary).<br>install one e.g. by running <strong>apt-get install wakeonlan</strong> or <strong>apk add awake</strong> (on the server)");//if non of the apps from the array are installed show a warning
```

Comment:
Please adjust the grammar as follows:
`A suitable WOL application ($apps) could not be found. Please install one.`

Also, consider patching the new setup script in this v3_beta branch to install these so taht this functionality is purely a fallback.